tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -108,6 +112,7 @@ public VerifiableConsumer(KafkaConsumer<String, String> consumer,
         this.useAutoCommit = useAutoCommit;
         this.useAsyncCommit = useAsyncCommit;
         this.verbose = verbose;
+        this.log = LoggerFactory.getLogger(VerifiableConsumer.class);
```

Comment:
If not using log context, the usual way is to have a `private static final` field for the logger.